test/abort/test-abort-aliased-buffer-overflow.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+const common = require('../common');
+const assert = require('assert');
+const cp = require('child_process');
+
+// This test ensures that during resizing of an Aliased*Array the computation
+// of the new size does not overflow.
+
+if (process.argv[2] === 'child') {
+  // test
+  const binding = require(`./build/${common.buildType}/binding`);
+
+  binding.allocateAliasedArrayBuffer();
+  const bigValue = BigInt('0xE000 0000 0000 0000');
```

Comment:
Fwiw, I’m not sure that this is going to work as expected on 32-bit platforms (because this value, converted to `size_t`, will be truncated to be `0`)

Reply:
Thank you! I have adapted the code so the value will be truncated to `0xE000 0000` for 32-bit.